src/components/embedded/EmbeddedMessage.tsx

```diff
@@ -27,6 +29,13 @@ function EmbeddedMessage({ messageId }: { messageId: string }) {
 	const [rawViewType, setRawViewType] = useState(MessageViewType.ASCII);
 	const [isExpanded, setIsExpanded] = React.useState(true);
 
+	const wrapperWidth = useElementSize(document.body as HTMLDivElement)?.width;
```

Comment:
body isnt HTMLDivElement. useElementSize should paramater should be of type HTMLElement.
and lets use ref for this one aswell. querying document for elements isnt react way to do it